src/bundle/Resources/config/prepend/ezpublish.yaml

```diff
@@ -298,14 +298,12 @@ system:
                     tr:
                         buttons:
                             ezmoveup:
-                                priority: 70
-                            ezmovedown:
                                 priority: 60
-                            ezattributesedit:
+                            ezmovedown:
                                 priority: 50
```

Comment:
I don't think we should change the priority of existing buttons, this is kind of a BC break.

Reply:
@dew326 fixed in https://github.com/ezsystems/ezplatform-richtext/pull/170/commits/135a88a81a3826621715fd188fd8f5771a79ef84